tests/test_faster_rcnn.py

```diff
@@ -254,16 +244,15 @@ def test_faster_rcnn_log_metric_val(mock_create, config, dataset):
         kfp_writer=kfp_writer,
     )
     epoch = 0
-    estimator.log_metric_val(dataset.label_mappings, epoch)
+    estimator.log_metric_val({"1": "car", "2": "bike"}, epoch)
 
```

Comment:
Why we are creating new object {"1": "car", "2": "bike"} when we have same in dataset?

Reply:
If you look at the test what is needed for the test is not the dataset but the label_mappings. Therefore i refactored the test to no longer include the dataset as an argument